src/Between.php

```diff
@@ -162,6 +179,13 @@ public function setInclusive($inclusive)
      */
     public function isValid($value)
     {
+        if ($this->numeric && ! is_numeric($value)) {
+            return false;
+        }
+        if (! $this->numeric && ! is_string($value)) {
+            return false;
+        }
```

Comment:
Each of these should be calling `$this->error()` in order to set a validation failure message; otherwise, there's no way to understand _why_ validation failed. Take a look at how the other cases that result in failure work, if you are unsure how to proceed.